### ISSUE TITLE
Server returned 0 (HERR)

### ISSUE THREAD

**Issue by as-com** [ArchiveTeam/soundcloud-grab#2]
```
Starting CheckIP for Item 
Finished CheckIP for Item 
Starting GetItemFromTracker for Item 
Received item 'api:59496300-59496309' from tracker
Starting PrepareDirectories for Item api:59496300-59496309
Finished PrepareDirectories for Item api:59496300-59496309
Starting WgetDownload for Item api:59496300-59496309
1=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
2=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
3=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
4=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
5=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
6=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
7=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
8=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
9=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
10=0 https://api.soundcloud.com/app/v2/tracks/59496300/comments?threaded=1&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=200&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.

I give up...
11=0 https://api-v2.soundcloud.com/tracks/59496300/related?anon_user_id=33006123&variant=control&client_id=2t9loNQH90kzJcsFCODdigxfp325aq4z&limit=10&offset=0&linked_partitioning=1&app_version=1500299175  
Server returned 0 (HERR). Sleeping.
```

```
andrew@newton /data/archiveteam/soundcloud-grab $ ./wget-lua --version
GNU Wget 1.14.lua.20160530-955376b built on linux-gnu.

+digest +https +ipv6 +iri +large-file -nls -ntlm +opie +ssl/gnutls 

Wgetrc: 
    /usr/local/etc/wgetrc (system)
Compile: gcc -DHAVE_CONFIG_H -DSYSTEM_WGETRC="/usr/local/etc/wgetrc" 
    -DLOCALEDIR="/usr/local/share/locale" -I. -I../lib -I../lib 
    -I/usr/include/lua5.1 -O2 -Wall 
Link: gcc -O2 -Wall -lgnutls -lgcrypt -lgpg-error -lz -lz -lidn -lpcre -lm 
    -ldl -llua5.1 ftp-opie.o gnutls.o ../lib/libgnu.a 

Copyright (C) 2011 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later
<http://www.gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Originally written by Hrvoje Niksic <hniksic@xemacs.org>.
Please send bug reports and questions to <bug-wget@gnu.org>.
andrew@newton /data/archiveteam/soundcloud-grab $ ~/.local/bin/run-pipeline --version
0.9.2
andrew@newton /data/archiveteam/soundcloud-grab $ python --version
Python 2.7.12
andrew@newton /data/archiveteam/soundcloud-grab $ uname -a
Linux newton 4.8.0-58-generic #63~16.04.1-Ubuntu SMP Mon Jun 26 18:08:51 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
andrew@newton /data/archiveteam/soundcloud-grab $ lsb_release -a
LSB Version:	core-9.20160110ubuntu0.2-amd64:core-9.20160110ubuntu0.2-noarch:security-9.20160110ubuntu0.2-amd64:security-9.20160110ubuntu0.2-noarch
Distributor ID:	LinuxMint
Description:	Linux Mint 18.2 Sonya
Release:	18.2
Codename:	sonya
```

**Comment by B13rg** [comment]
When this happens it seems that soundcloud is blocking you.  A fix that worked for some other people on the irc channel was changing the USER_AGENT in pipeline.py

**Comment by as-com** [comment]
@B13rg Yep, setting the user agent to a "legitimate" browser worked.

**Comment by Arkiver2** [comment]
This should be fixed with 4df8a79d201c60302d9b1df02459b6c61b88733d.
